src/main/flight/gps_rescue.c

```diff
@@ -125,6 +125,8 @@ PG_RESET_TEMPLATE(gpsRescueConfig_t, gpsRescueConfig,
     .angle = 32,
     .initialAltitudeM = 50,
     .descentDistanceM = 200,
+    .targetLandingAltitudeCm = 700,
```

Comment:
This seems to be in centimeter units but the parameter in settings.c implies meters.

Reply:
yes, it is in centimeter, sorry but I don't get where is implied to be in meters in settings.c

Reply:
The settings parameter has no indicator units and all the other distance/altitude parameters are in meters.

Reply:
I can convert it to meters too or change its name to ` gps_rescue_landing_alt_cm`, which solution do you prefer?

Reply:
It makes more sense for the parameter to be in meters - can't see a need where the user would need to specify fractions of a meter. Internally you can convert to and use centimeters if needed.

Reply:
Done